### PR TITLE
[igemm]fix tuning paramter of spare(issue 598)

### PR DESCRIPTION
**Pull request by asleepzzz**
fix issue #598
we remove spare set judgement in is_fast function
due to we need fully searching it when 2nd round

### CODE REVIEW

File: src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp [hunk 1]
```diff
@@ -392,6 +392,8 @@ bool PerformanceImplicitGemmBwdDataV4R1Xdlops::IsReallyValid(const ConvolutionCo
 bool PerformanceImplicitGemmBwdDataV4R1Xdlops::IsFastToBeUsedForTuning(
     const ConvolutionContext& ctx) const
 {
+    if(use_spare_set)
+        return true;
```
Comment (atamazov) [inline src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp:396]: See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/605#discussion_r527178430

**Review by atamazov — CHANGES_REQUESTED** [review]

File: src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp (hunk 1 above)
Comment (asleepzzz) [reply src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp:396]: let @zjing14 and we discuss it

Comment (atamazov) [reply src/solver/conv_hip_implicit_gemm_bwd_data_v4r1_xdlops.cpp:396]: [Recommendation] See https://github.com/ROCmSoftwarePlatform/MIOpen/pull/605#discussion_r528903383.

Resolved.

**Review by atamazov — APPROVED** [review]
LGTM

**Review by atamazov — APPROVED** [review]

**Comment by asleepzzz** [comment]
change PR to 644,close this

**Pull request closed without merging**